src/constants.js

```diff
@@ -1,10 +1,33 @@
 module.exports = {
+  RouterPath: {
```

Comment:
do we use these anywhere else? I personally find it more confusing to have to go to this file and see what is going on than explicitly putting this where it's being used. @dwalintukan